nginx-hello-nonroot/plain-text-version/Makefile

```diff
@@ -0,0 +1,15 @@
+VERSION=0.1
+PREFIX=nginxdemos/nginx-hello
+TAG=$(VERSION)
```

Comment:
the tag should be `$(VERSION)-plain-text`.
Make sure to incorporate suggestions for the html version makefile as well.